llvm/utils/TableGen/CompressInstEmitter.cpp

```diff
@@ -123,7 +124,7 @@ class CompressInstEmitter {
   const RecordKeeper &Records;
   const CodeGenTarget Target;
   std::vector<CompressPat> CompressPatterns;
-
+  unsigned SourceLastTiedOp; // postion of the last tied operand in Source Inst
```

Comment:
No other class members are storing a state for single instruction. Is this just a temporary variable between 2 different functions while processing a single instruction? Can we just make it a reference argument to the relevant function that needs to modify it?